### PR TITLE
Include serde_derive

### LINKED ISSUE #82: serde_derive
Not sure if it's against the protocol, but I'd vouch for serde_derive to be added to the top crates.
I wanted to add a modern playground example to http://stackoverflow.com/a/37565736/257568 but it turns out there's no serde_derive there yet.

I mean, Serde is certainly in the top crates, but we'd want to promote the newer serde_derive version of the automation instead of the older serde_macros, right?

### PR DESCRIPTION
**Pull request by dtolnay**
Fixes #82.

### CODE REVIEW

File: top-crates/Cargo.toml [hunk 1]
```diff
@@ -4,14 +4,11 @@ name = "top-crates"
...
+lazy_static = "0.2"
 serde = "0.9.6"
+serde_derive = "0.9"
 serde_json = "0.9.5"
-
-[dependencies.toml]
-version = "0.2.0"
-default-features = false
-features = ["serde"]
+toml = { git = "https://github.com/alexcrichton/toml-rs" }
```
Comment (dtolnay) [inline top-crates/Cargo.toml]: Serde 0.9 support exists on the master branch, update to 0.3 once it is available. https://github.com/alexcrichton/toml-rs/pull/137#issuecomment-278647851

**Comment by shepmaster** [comment]
Oh dear, I've already written most / all of this locally; I certainly didn't expect you to do so! I'll give this the once-over and see what good bits I can steal 😉

**Comment by shepmaster** [comment]
I do like the idea of relying on Cargo though, would allow me to nicely sidestep alexcrichton/toml-rs#142

**Comment by dtolnay** [comment]
Is there anything you are working on including in your implementation that I don't have here? If you don't have time to work on this, I can make the changes here.

**Comment by shepmaster** [comment]
@dtolnay mostly just disagreements with commit sizing / grouping, so I'm in the process of splitting up your commit. I expect to make progress this evening (Eastern).

**Comment by shepmaster** [comment]
Closed via #100; thanks so much!

**Pull request closed without merging**